huaweicloud/services/meeting/resource_huaweicloud_meeting_conferences.go

```diff
@@ -0,0 +1,1134 @@
+						},
+						"is_send_sms": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"is_send_calendar": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"is_auto_mute": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"is_hard_terminal_auto_mute": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"is_guest_free_password": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"callin_restriction": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.IntInSlice([]int{
+								int(CallInTypeAll), int(CallInTypeEnterprise), int(CallInTypeInvitee),
+							}),
+						},
+						"allow_guest_start": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+						"guest_password": {
+							Type:      schema.TypeString,
+							Optional:  true,
+							Computed:  true,
+							Sensitive: true,
+						},
+						"prolong_time": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							Computed:     true,
+							ValidateFunc: validation.IntBetween(0, 60),
+						},
+						"waiting_room_enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"conference_type": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"access_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"chair_join_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"guest_join_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"audience_join_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"subconferences": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     subconferenceSchemaResource(),
+			},
+			"join_password": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"host": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"guest": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func buildConferenceParticipants(participants []interface{}) []conferences.Participant {
+	if len(participants) < 1 {
+		return nil
+	}
+
+	result := make([]conferences.Participant, len(participants))
+	for i, val := range participants {
+		participant := val.(map[string]interface{})
+		result[i] = conferences.Participant{
+			UserUUID:     participant["user_id"].(string),
+			AccountId:    participant["account_id"].(string),
+			AppId:        participant["name"].(string),
```

Comment:
wrong field setting, it's name
and in L751, you get the name from `participant.Name`

Reply:
done.